src/execution/service.ts

```diff
@@ -8,23 +8,27 @@ import {BaseExecution} from './base.js';
 import {Fingerprint} from '../fingerprint.js';
 
 import type {ExecutionResult} from './base.js';
-import type {Executor} from '../executor.js';
 import type {ServiceScriptConfig} from '../config.js';
 import type {Logger} from '../logging/logger.js';
+import type {Executor} from '../executor.js';
 
 /**
  * Execution for a {@link ServiceScriptConfig}.
  */
 export class ServiceScriptExecution extends BaseExecution<ServiceScriptConfig> {
   static execute(
-    config: ServiceScriptConfig,
+    script: ServiceScriptConfig,
```

Comment:
did you mean to go back to `script`?

Reply:
Oops, no, merge mistake. But this whole method is removed in the next commit anyway so no point fixing.